README.md

```diff
@@ -53,15 +53,15 @@ app.post('/cool-profile', cpUpload, function (req, res, next) {
 })
 ```
 
-In case you need to handle a text-only multipart form, you can use any of the multer methods (`.single()`, `.array()`, `fields()`). Here is an example using `.array()`:
+In case you need to handle a text-only multipart form, you should use `none()` method:
```

Comment:
English is not my first language, but I think that there should be a "the" in there. Also, could you add a `.` before `none()`?

> In case you need to handle a text-only multipart form, you should use the `.none()` method: